FBSnapshotTestCase/FBSnapshotTestCasePlatform.m

```diff
@@ -34,7 +34,7 @@ BOOL FBSnapshotTestCaseIs64Bit(void)
 NSString *FBDeviceAgnosticNormalizedFileName(NSString *fileName)
 {
   UIDevice *device = [UIDevice currentDevice];
-  CGSize screenSize = [[UIApplication sharedApplication] keyWindow].bounds.size;
```

Comment:
@Grubas7 the short-circuit is intentional, and our app is already the "Host" for our tests.  This removes the unnecessary "link in both targets" step when adding new app code.

My question to you guys is, should FBSnapshotTestCase be defensive by setting the `keyWindow` itself (which it already tries to do), and if so, should it guarantee that `keyWindow